.github/release-drafter-config.yml

```diff
@@ -0,0 +1,23 @@
+# Configuration for Release Drafter: https://github.com/toolmantim/release-drafter
+template: |
```

Comment:
Please refer to Alerting plugin https://github.com/opendistro-for-elasticsearch/alerting/blob/master/.github/draft-release-notes-config.yml so that we can have consistency across the release notes.